.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E203, E266, E501, W503, F403, F401, E231
+ignore = E203, E266, E501, W503, F403, F401, E231, W503
```

Comment:
Is dit een workaround voor een bug oid? Er staat nu 2x `W503`

Reply:
Scherp!!